consensus/reactor_test.go

```diff
@@ -131,6 +132,62 @@ func TestValidatorSetChanges(t *testing.T) {
 	// TODO: test more changes!
 }
 
+// Check we can make blocks with skip_timeout_commit=false
+func TestReactorWithTimeoutCommit(t *testing.T) {
```

Comment:
I don't think this does what we want since it only creates 3 validators. We want 4 validator, where eg one of them is offline. So let's copy `TestReactor` with N=4 but use `N-1` in the for loop, the MakeConnectedSwitches, and the timeoutWaitGroup.